### PR TITLE
DEV: Fix Flake8 Errors and Typos in `models/engine/patchpredictor.py`

### PR DESCRIPTION
**Pull request by shaneahmed**
- Fix Flake8 errors and Typos in `models/engine/patchpredictor.py`

### CODE REVIEW

**Comment by shaneahmed** [comment]
@John-P This cannot pass both deepsource and flake8. The ignore flags cannot be added for back slash for both.

**Comment by vqdang** [comment]
Will review once it pas travis.

File: tiatoolbox/models/engine/patch_predictor.py [hunk 1]
```diff
@@ -452,6 +452,289 @@ def _predict_engine(
 
         return cum_output
 
+    def _predict_patch(self, imgs, labels, return_probabilities, return_labels, on_gpu):
```
Comment (vqdang) [inline tiatoolbox/models/engine/patch_predictor.py:455]: Move to be above the `_predict_tile_wsi`

Comment (shaneahmed) [reply tiatoolbox/models/engine/patch_predictor.py]: Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/365/commits/fb2c52ef882a0e90fe0427d00c617073852b01ff

**Comment by shaneahmed** [comment]
@vqdang This is ready for review.

**Review by vqdang — APPROVED** [review]

**Pull request merged**